main.js

```diff
@@ -0,0 +1,19 @@
+const allPosts= document.querySelector("li");
+allPosts.remove();
+// * removed allposts *//
+
+const readTime =document.querySelector("span");
+readTime.remove();
+//* removed readtime *//
+
+const articles =document.querySelectorAll("article");
+let test;
```

Comment:
How can you use knowing that the last index is one less than the length of the array to help you out here?